syntaxes/omt.tmLanguage.json

```diff
@@ -649,7 +657,14 @@
             "patterns": [
                 { "include": "#comments" },
                 {
-                    "begin": "^( +)(\\S+):(.*)$",
+                    "match": "^ +(\\S+): +\\[](.*)$",
+                    "captures": {
+                        "1": { "name": "support.type.module.omt" },
+                        "2": { "patterns": [{ "include": "#_comments-only" }] }
+                    }
+                },
+                {
+                    "begin": "^( +)(\\S+):(?! +\\[])(.*)$",
```

Comment:
why capture the first group if you are not going to use it?

- [ ] remove the first capture group

Reply:
The first group is captured because it is used for recognising the end of this block. Since the end regex didn't change, it is easily missed in this diff view ;)